src/index.js

```diff
@@ -27,9 +27,6 @@ cleared.addEventListener('click', () => {
   newTodo.clearCompleted();
 });
 
-cleared.addEventListener('click', () => {
-  newTodo.clearCompleted();
-});
 for (let i = 0; i < statusInput.length; i += 1) {
   statusInput[i].addEventListener('change', (ev) => {
     newTodo.changeStatus(ev.target.id, ev.target.checked);
```

Comment:
- [ ]  I can see that you have already implemented almost all the tasks for upcoming projects 🤔. Currently, the local storage does not get updated when the checkbox is changed. The required behavior is that the local storage gets updated and toggles completed from false to true and also puts a line through the completed task. Please kindly fix this by updating the local storage when there is a toggle of the checkbox and putting a line through the completed task. Please check the project requirement [here](https://www.youtube.com/watch?v=AcUd-_Yjjqg)

<img width="827" alt="chrome_ULi3BxHR31" src="https://user-images.githubusercontent.com/81468675/154543817-9b60d665-d892-4387-951e-d249fbc411ca.png">